sspl_test/messaging/ingress_processor_tests.py

```diff
@@ -219,16 +220,18 @@ def _process_msg(self, body):
             logger.exception(
                 "_process_msg unrecognized message: %r" % ingressMsg)
 
-    def _read_config(self):
+    def _init_config(self):
         """Configure the messaging exchange with defaults available."""
         # Make methods locally available
-        self._node_id = Conf.get(SSPL_TEST_CONF, NODE_ID_KEY, 'SN01')
+        self._node_id = Conf.get(GLOBAL_CONF, NODE_ID_KEY, DEFAULT_NODE_ID)

```

Comment:
Here also you can use get_current_node_id().

Reply:
Tried this earlier but it was failing with import error. 
```
File "/opt/seagate/cortx/sspl/sspl_test/run_test.py", line 33, in <module>
from common import (TestFailed, init_messaging_msg_processors,
File "/opt/seagate/cortx/sspl/sspl_test/common.py", line 37, in <module>
from messaging.ingress_processor_tests import IngressProcessorTests 
File "/opt/seagate/cortx/sspl/sspl_test/messaging/ingress_processor_tests.py", line 37, in <module>
from common import get_current_node_id
ImportError: cannot import name 'get_current_node_id'
```